actions/canvas_send_image_MOD.js

```diff
@@ -130,7 +130,7 @@ module.exports = {
     const buffer = canvas.toBuffer('image/png', { compressionLevel: compress });
     const attachment = new DiscordJS.MessageAttachment(buffer, name);
     if (target && target.send) {
-      target.send(this.evalMessage(data.message, cache), attachment).then((msgobject) => {
+      target.send({ content: this.evalMessage(data.message, cache), files: [attachment] }).then((msgobject) => {
```

Comment:
Is the `message` option supposed to be optional? If so, you need to account for that, since `this.evalMessage()` will return an empty string, which will cause discord.js to throw